src/plugins/share/public/components/modals/embed_modal_page.tsx

```diff
@@ -6,37 +6,39 @@
  * Side Public License, v 1.
  */
 
-import React, { Component, ReactElement } from 'react';
 import {
-  EuiButton,
-  EuiCopy,
+  EuiForm,
```

Comment:
nit: I would have expected this file to be called `embed_modal.tsx` without the `_page` as it contains the `EmbedModal` component.

Reply:
The same for the `LinkModal`.